logic/overworld.py

```diff
@@ -209,7 +209,10 @@ def __init__(self, options, world_setup, r):
         bay_water.connect(ukuku_prairie, FLIPPERS)
         bay_water.connect(left_bay_area, FLIPPERS)
         fisher_under_bridge = Location().add(TradeSequenceItem(0x2F5, TRADING_ITEM_NECKLACE))
-        fisher_under_bridge.connect(bay_water, AND(TRADING_ITEM_FISHING_HOOK, FEATHER, FLIPPERS))
+        if options.logic == "casual":
+            fisher_under_bridge.connect(bay_water, AND(TRADING_ITEM_FISHING_HOOK, FEATHER, FLIPPERS))
+        else
+            fisher_under_bridge.connect(bay_water, AND(TRADING_ITEM_FISHING_HOOK, OR(FEATHER, SWORD, BOW), FLIPPERS)) # just swing/shoot at fisher, trivially easy
```

Comment:
@daid should i take this part out and just use `AND(TRADING_ITEM_FISHING_HOOK, FEATHER, FLIPPERS)`? i think sword/bow are dirt easy to do, and the differentiation between doing itemless is nice, but this isn't exactly "intended behavior" for normal logic, in retrospect, so i thought i should bring it up.

Reply:
so, how does this work with bow? I'm not sure about this one yet, I've seen that you have been experimenting, but I haven't completely followed it (with me being sick and all)

Reply:
> so, how does this work with bow? I'm not sure about this one yet, I've seen that you have been experimenting, but I haven't completely followed it (with me being sick and all)

When you fire off a shot, for whatever reason, link's hitbox for triggers expands in front of him, and it makes it really easy to talk to the fisherman. I figure that it's due to having to check for bomb arrows. I get it first or second try with the bow: you just have to go directly underneath the fisherman, and shoot the bow while it's bound to the A button.